Assets/Scripts/DeathZone.cs

```diff
@@ -0,0 +1,14 @@
+﻿    using System;
+    using Events;
+    using UnityEngine;
+
+    //If player lands enters a collider of dead zone, they die
+    [RequireComponent(typeof(BoxCollider))]
+    public class DeathZone : MonoBehaviour
```

Comment:
Not an issue for now, but should try and give it a namespace and put it in a sub-folder for scripts.